### PR TITLE
Upgrade Robot fixtures

### PR DESCRIPTION
**Pull request by abrooksv**
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt [hunk 1]
```diff
@@ -7,53 +7,185 @@ import com.intellij.remoterobot.RemoteRobot
...
+    private fun findExpandedPath(vararg path: String, fullMatch: Boolean): TreePathToRow? = collectExpandedPaths().singleOrNull { expandedPath ->
+        expandedPath.path.size == path.size && expandedPath.path.containsAllNodes(*path, fullMatch = fullMatch) ||
+            expandedPath.path.size - 1 == path.size && expandedPath.path.drop(1).containsAllNodes(*path, fullMatch = fullMatch)
+    }
+
+    private fun findExpandedRowWithText(text: String, fullMatch: Boolean = true): TreePathToRow? = collectExpandedPaths().singleOrNull { expandedPath ->
+        if (fullMatch) expandedPath.path.last() == text else expandedPath.path.last().contains(text, true)
+    }
+
+    private fun List<String>.containsAllNodes(vararg treePath: String, fullMatch: Boolean): Boolean = zip(treePath).all {
```
Comment (hunterwerlla) [inline ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt:177]: lots of things have changed here, but what is added to this newer version on top of what is forked?

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/FileBrowserFixture.kt [hunk 2]
```diff
@@ -40,9 +40,7 @@ class FileBrowserFixture(
     remoteComponent: RemoteComponent
 ) : DialogFixture(remoteRobot, remoteComponent) {
     private val tree by lazy {
-        find<JTreeFixture>(byXpath("//div[@class='Tree']"), Duration.ofSeconds(10)).also {
-            it.separator = "|" // switch out the separator since tree has path "/" as the root
```
[the remark this answers is not in the document]
Comment (abrooksv) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/FileBrowserFixture.kt:44]: Their JTree avoid AssertJSwing to do expansion

File: ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt (hunk 1 above)
Comment (abrooksv) [reply ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/JTreeFixture.kt:195]: I'll add comments, its the JS blobs

heres the patch: https://github.com/JetBrains/intellij-ui-test-robot/issues/38#issuecomment-840973361

**Review by hunterwerlla — APPROVED** [review]

**Review by rli — APPROVED** [review]

**Review by rli — APPROVED** [review]

**Pull request merged**